GDJS/Runtime/AsyncTasksManager.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2022 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  /**
+   * A manager for all asynchronous tasks.
+   * @see {@link RuntimeScene.getAsyncTasksManager}.
+   */
+  export class AsyncTasksManager {
+    /**
+     * Maps a task to the callback to be executed once it is finished.
+     */
+    private tasks = new Array<{
```

Comment:
Maybe name it `tasksWithCallback` (I can't find a better name). Seeing this task.asyncTask below seemed weird to me